sources/api/migration/migrator/src/main.rs

```diff
@@ -82,11 +124,65 @@ fn run() -> Result<()> {
             process::exit(0);
         });
 
-    let migrations = find_migrations(
-        &args.migration_directories,
-        &current_version,
-        &args.migrate_to_version,
-    )?;
+    // DEPRECATED CODE BEGIN ///////////////////////////////////////////////////////////////////////
+    // check if the `from_version` supports signed migrations. if not, run the 'old'
+    // unsigned migrations code and return.
+    if !are_migrations_signed(&current_version) {
+        // note in the system journal that the unsigned code path ran.
+        eprintln!("migrator is running unsigned migrations");
```

Comment:
@tjkirch left a comment here that, inexplicably, GitHub allowed me to delete when I was, in fact, trying to delete my own response. Fortunately I have both pasted to a text file:

@tjkirch wrote:
> I don't think we need these messages in the logs/journal, we know which type of migrations ran based on the version. (Ties with my other comments about simplifying/removing the branching.)

And I replied:
I am using them during test cycles. I can delete them, but once I do I will no longer be able to verify correctness during the test cycles as I have been (see the test procedure in the PR comments).

Reply:
I'd prefer to use a more structured method like running different migrations on each path.  I'm hoping we can go back to the shell script method so having test migrations is trivial...

Reply:
I meant when I run Bottlerocket and check the logs to ensure that migrator did what I want. However, I can remove these print statements as the last push.

Reply:
Yeah, I was suggesting that we use different test migrations (which can print different output) to get the same effect, and even check it in the test, rather than impacting the actual codebase here.

Reply:
removed these in this push https://github.com/bottlerocket-os/bottlerocket/compare/b6fe3cd824df6df149da51efc14b01b222f46787..7b6b6c27a0abbeedd77535ba948aa0c7d220bb1c